server/src/main/resources/templates/web/eastwest/login.html

```diff
@@ -41,6 +41,21 @@ <h4 class="modal-title">This is a U.S. government service.</h4>
 
 
         <div class="js-full_login">
+
+        <div class="alert alert-danger">
```

Comment:
I suggest indenting this whole block for readability (specifically, so the nesting is easier to parse).